iothub/device/src/ModernDotNet/HsmAuthentication/Transport/HttpUdsMessageHandler.cs

```diff
@@ -20,8 +20,8 @@ public HttpUdsMessageHandler(Uri providerUri)
 
         protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
-            Socket socket = await GetConnectedSocketAsync().ConfigureAwait(false);
-            HttpBufferedStream stream = new HttpBufferedStream(new NetworkStream(socket, true));
+            using Socket socket = await GetConnectedSocketAsync().ConfigureAwait(false);
+            using var stream = new HttpBufferedStream(new NetworkStream(socket, true));
```

Comment:
Q - does the returned `HttpResponseMessage` contain a reference to the `HttpBufferedStream`? If yes, then we shouldn't dispose it here.